rivera_diaz-A/PrincipiosSOLID-SolucionTarea/README.md

```diff
@@ -0,0 +1,17 @@
+# Practica 2: Principios SOLID: Ejemplo Parqueadero Público
+## Realizado en Java - Maven
+
+Con el fin de realizarse las pruebas unitarias se debera seguir los siguientes pasos:
+
+## Posicionarse en la ubicacion del proyecto: 
+```
+cd ...\labisii2021.2\rivera_diaz\Practica_1
+```
+## Para construir el .jar: 
+```
+mvn package 
+```
```

Comment:
NO corre el programa:

java -cp target/PrincipiosSOLID-SolucionTarea-1.0-SNAPSHOT.jar co.unicauca.parkinglot.presentation.ClienMain
Nov 18, 2021 12:01:53 PM co.unicauca.parkinglot.access.VehicleRepository connect
SEVERE: null
java.sql.SQLException: No suitable driver found for jdbc:sqlite::memory:
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:702)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at co.unicauca.parkinglot.access.VehicleRepository.connect(VehicleRepository.java:98)
	at co.unicauca.parkinglot.access.VehicleRepository.initDatabase(VehicleRepository.java:83)
	at co.unicauca.parkinglot.access.VehicleRepository.<init>(VehicleRepository.java:26)
	at co.unicauca.parkinglot.access.RepositoryFactory.<init>(RepositoryFactory.java:14)
	at co.unicauca.parkinglot.access.RepositoryFactory.getInstance(RepositoryFactory.java:19)
	at co.unicauca.parkinglot.presentation.ClienMain.main(ClienMain.java:27)

Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.sql.Connection.createStatement()" because "this.conn" is null
	at co.unicauca.parkinglot.access.VehicleRepository.initDatabase(VehicleRepository.java:84)
	at co.unicauca.parkinglot.access.VehicleRepository.<init>(VehicleRepository.java:26)
	at co.unicauca.parkinglot.access.RepositoryFactory.<init>(RepositoryFactory.java:14)
	at co.unicauca.parkinglot.access.RepositoryFactory.getInstance(RepositoryFactory.java:19)
	at co.unicauca.parkinglot.presentation.ClienMain.main(ClienMain.java:27)